common/service/dynamicconfig/constants.go

```diff
@@ -308,7 +309,8 @@ const (
 	ShardUpdateMinInterval
 	// ShardSyncMinInterval is the minimal time interval which the shard info should be sync to remote
 	ShardSyncMinInterval
-
+	// EventEncodingPropertyFilteredByDomainKey is the encoding type for history events
+	EventEncodingPropertyFilteredByDomainKey
```

Comment:
Let's stick to the convention used by other properties and call it DefaultEventEncoding or something.

Reply:
I will talk to Bowei to make sure this sticks to our convention

Reply:
We don't need surfix `PropertyFilteredByDomainKey` in config name. Agree with samar just name it `EventEncoding` or `DefaultEncoding` etc.

Reply:
I see. Suffix is only a convention in config function.